db/models.py

```diff
@@ -50,3 +59,64 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(to=settings.AUTH_USER_MODEL,
+                             on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(to=MovieSession,
+                                      on_delete=models.CASCADE)
+    order = models.ForeignKey(to=Order, on_delete=models.CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        constraints = [
+            UniqueConstraint(
+                fields=["row", "seat", "movie_session"],
+                name="uniq_ticket"
+            )
+        ]
+
+    def __str__(self) -> str:
+        return (f"{self.movie_session} "
+                f"(row: {self.row}, seat: {self.seat})")
+
+    def clean(self) -> None:
+        if not 1 < self.row <= self.movie_session.cinema_hall.rows:
+            raise ValidationError(
+                {"row": [f"row number must be in available range: "
+                         f"(1, rows): (1, "
+                         f"{self.movie_session.cinema_hall.rows})"]}
+            )
+        if not 1 < self.seat <= self.movie_session.cinema_hall.seats_in_row:
+            raise ValidationError(
+                {"seat": [f"seat number must be in available range: "
+                          f"(1, seats_in_row): (1, "
+                          f"{self.movie_session.cinema_hall.seats_in_row})"]}
+            )
```

Comment:
You could avoid code duplication here because there are several repetitive parts in this method.